src/snowflake/cli/api/cli_global_context.py

```diff
@@ -386,5 +386,19 @@ def _should_force_mute_intermediate_output(self) -> bool:
         return self._manager.output_format == OutputFormat.JSON
 
 
-cli_context_manager: _CliGlobalContextManager = _CliGlobalContextManager()
-cli_context: _CliGlobalContextAccess = _CliGlobalContextAccess(cli_context_manager)
+_CLI_CONTEXT_MANAGER: _CliGlobalContextManager | None = None
+_CLI_CONTEXT: _CliGlobalContextAccess | None = None
+
+
+def get_cli_context_manager() -> _CliGlobalContextManager:
```

Comment:
I am curious, what is the reason for this change? (Would help to add it in the PR description as well)

Reply:
Added